enrolment-ui/src/index.js

```diff
@@ -30,37 +32,40 @@ unregister();
 
 // Create redux store
 const store = configureStore();
+const { clientId } = getOneConfig('clientId');
 
 render(
   <Provider store={store}>
-    <BrowserRouter>
-      <App>
-        <Switch>
-          <OnlyNonLoggedInRoute exact path="/" component={Login} />
-          <OnlyNonLoggedInRoute exact path="/register-user/:projectId/:userId" component={Login} />
-          <OnlyNonLoggedInRoute exact path="/login" component={Login} />
-          <Main>
-            <Switch>
-              <PrivateRoute exact path="/dashboard" component={Dashboard} />
-              <PrivateRoute exact path="/register/project" component={RegisterProject} />
-              <PrivateRoute exact path="/view/project/:id" component={RegisterProject} />
-              <PrivateRoute
-                exact
-                path="/register/user/:projectId/:userId"
-                component={RegisterUser}
-              />
-              <PrivateRoute
-                exact
-                path="/view/project-user/:projectId/:userId"
-                component={RegisterUser}
-              />
-              <PrivateRoute exact path="/projects" component={Projects} />
-              <PrivateRoute component={NotFound} />
-            </Switch>
-          </Main>
-        </Switch>
-      </App>
+    <GoogleOAuthProvider clientId={clientId}>
```

Comment:
Looks like a lot, but I'm just wrapping the App in the `GoogleOAuthProvider` here. Necessary for the new Google auth library.